mmf/configs/datasets/localized_narratives/defaults.yaml

```diff
@@ -0,0 +1,49 @@
+dataset_config:
+  localized_narratives:
+    data_dir: ${env.data_dir}/datasets
+    depth_first: false
+    fast_read: false
+    use_images: false
+    use_features: true
+    zoo_requirements:
+    - localized_narratives.defaults
+    - flickr30k.defaults
+    - ade20k.defaults
+    trace_mode: true
+    features:
+      train:
+      - flickr30k/defaults/features/detectron.lmdb
+      - ade20k/defaults/features/ade20k_train.lmdb
+      val:
+      - flickr30k/defaults/features/detectron.lmdb
+      - ade20k/defaults/features/ade20k_val.lmdb
+      test:
+      - flickr30k/defaults/features/detectron.lmdb
+    annotations:
+      train:
+      - localized_narratives/defaults/annotations/flickr30k_train_localized_narratives.jsonl
+      - localized_narratives/defaults/annotations/ade20k_train_localized_narratives.jsonl
+      val:
+      - localized_narratives/defaults/annotations/flickr30k_val_localized_narratives.jsonl
+      - localized_narratives/defaults/annotations/ade20k_validation_localized_narratives.jsonl
+      test:
+      - localized_narratives/defaults/annotations/flickr30k_test_localized_narratives.jsonl
+    max_features: 100
+    processors:
+      text_image_processor:
+          type: text_token_overlap_with_image_bbox
+          params:
+            trace_num_samples: 8
+            remove_punctuations: true
+            text_processor:
```

Comment:
Hmm, I see text_processor is part of this. This is somewhat weird, but maybe I will understand down the line.